### ISSUE TITLE
Use SilverStripe code comments

### ISSUE THREAD

**Issue by MattyBalaam** [aaroncdr/ss-atom#2]
This is a feature request.

At the moment when using a shortcut to comment code, Atom is using html comments 

Would it be possible to use Silverstripe comments instead which do not get output as html?  <%-- --%>

**Comment by ACDR** [comment]
Hey @MattyBalaam, there's currently a '-- ' snippet to create a silverstripe comment. Is that what you're looking for?

**Comment by MattyBalaam** [comment]
Not exactly, the snippet is fine for adding a new comment, but not as powerful as the ctrl-/ combo which wraps either a selection or the current line under the cursor in a comment.
